test/integration/event_page_test.rb

```diff
@@ -0,0 +1,62 @@
+require 'test_helper'
+
+feature 'Event' do
+  def setup
+    @user = make_user
+    @admin = make_admin
+  end
+
+  test 'shows a logo image if the event has a logo' do
+    sign_in_as_user
+    make_event(name: 'The Event', approved: true, logo: 'logo-image')
+
+    visit events_path
+
+    assert page.text.include?('The Event')
+    assert page.body.include?('logo-image')
+    assert_not page.body.include?('event-default')
+  end
+
+  test 'shows a default image if the event does not have a logo' do
+    sign_in_as_user
+    make_event(name: 'The Event', approved: true)
+
+    visit events_path
+
+    assert page.text.include?('The Event')
+    assert page.body.include?('event-default')
+  end
+
+  test 'shows Your events in the breadcrumb is the user is an organizer' do
+    sign_in_as_user
+    make_event(name: 'The Event', approved: true, organizer_id: @user.id)
+
+    visit events_path
+
+    click_link 'The Event'
+
+    assert page.text.include?('Your Events')
+  end
+
+  test 'shows Events in the breadcrumb is the user is not an organizer' do
+    sign_in_as_user
+    make_event(name: 'The Event', approved: true)
+
+    visit events_path
+
+    click_link 'The Event'
+
+    assert page.text.include?('Events')
```

Comment:
could it also mean that page may include 'Your Events' ?

Reply:
yes, I think we should be more specific in this test. right now we are checking that the word is present in the whole page's body instead of checking the breadcrumb. i would go for something like 'assert_select "a[href=?]", "/desired_path/1"'

Reply:
👍